client/marklogicClient.ts

```diff
@@ -486,14 +497,16 @@ export function requestMarkLogicUnitTest(
 }
 
 export function newMarklogicClientWithPort(
-    dbClientContext: ClientContext, targetPort: number, targetPath: string
+    dbClientContext: ClientContext,
+    targetPort: number,
+    targetPath: string
 ): ClientContext {
-    const manageClientParams: MlClientParameters = JSON.parse(JSON.stringify(dbClientContext.params));
-    manageClientParams.port = targetPort;
-    manageClientParams.restBasePath = targetPath;
-    manageClientParams.contentDb = null;
-    manageClientParams.modulesDb = null;
-    return new ClientContext(manageClientParams);
+    const newClientParams: MlClientParameters = JSON.parse(JSON.stringify(dbClientContext.params));
+    newClientParams.port = targetPort;
+    newClientParams.restBasePath = targetPath;
+    newClientParams.contentDb = null;
```

Comment:
I know this code was already here and this is just a rename, but do you know why `contentDb` and `modulesDb` are nulled out? I think that deserves a comment as it seems surprising - i.e. to a casual reader, I'm left wondering "Wait, the user may have configured these, so why are we now ignoring them?"